### DISCUSSION TITLE
Informations importantes - Test 2

### DISCUSSION THREAD

**Discussion by OlivierLmr** [web-classroom/web-classroom.github.io#18] [Announcements]
Bonsoir,

Pour le test de la semaine prochaine, voici quelques informations.

Le test portera sur les "chapitres" 5 à 9 inclus, donc tout ce qui a été vu depuis le premier test.

Aussi, j'ai décidé d'expérimenter une approche différente de l'autre fois. Par conséquent, vous serez autorisés à accéder aux ressources suivantes.
- Tout document papier de votre choix (par exemple un résumé fait par vos soins, ou volé à un ami, même si ce n'est pas dans votre intérêt), sans limite du nombre de page (dans les limites de la raison).
- Le site du cours.
- Les [MDN Web Docs](https://developer.mozilla.org/en-US/).
En revanche, vous ne pourrez pas vous servir d'un moteur de recherche, d'intelligence artificielle, ou d'outil de communication.

Notez que le test contiendra donc peu de questions "de cours" (du genre "quelle est la définition de ...") qui seraient maintenant trop impertinentes, mais plus de questions de cas particuliers à analyser (par exemple "qu'affiche le code suivant").

Il y aura par ailleurs plus de questions vous demandant de justifier vos réponses, et moins de QCM.

En vous souhaitant une bonne semaine, et à mardi prochain,

Olivier

**Comment by theookage** [comment]
Aurons nous aussi accès aussi aux liens qui sont inclus dans les slides du cours ?
Par exemple, il y a des liens vers dit repo github qui ont servi d'exemple pour le cours, ou des liens vers des docs

**Comment by OlivierLmr** [comment]
Oui, vous pourrez accéder à ces contenus aussi, effectivement.
